### PR TITLE
[DX-1278] Dashboard release notes 5.3.1

### PR DESCRIPTION
**Pull request by andyo-tyk**
### Preview Link
https://deploy-preview-4400--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.3/

___

## **Type**
documentation

___

## **Description**
- Added a new section for Tyk Dashboard 5.3.1 Release Notes, including detailed information on the release date, breaking changes, deprecations, upgrade instructions, release highlights, downloads, and a comprehensive changelog.
- Updated the document tags to include "5.3.1".
- Modified the Tyk OAS APIs Compatibility Caveats section to improve document navigation.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Addition of 5.3.1 Release Notes and Minor Adjustments</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md
<li>Added tags for version 5.3.1 in the document header.<br> <li> Introduced a new section for the 5.3.1 Release Notes, including <br>release highlights, breaking changes, deprecations, upgrade <br>instructions, downloads, and a detailed changelog.<br> <li> Adjusted the Tyk OAS APIs Compatibility Caveats section to include an <br>anchor for direct linking.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4400/files#diff-14b1be0b2206c9ee21cb1002b55361919932e84558f614c4342f05b8eb5431f8">+64/-2</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

___

## **Type**
Documentation

___

## **Description**
- Added a comprehensive new section for Tyk Dashboard 5.3.1 Release Notes, detailing release highlights, breaking changes, deprecations, upgrade instructions, downloads, and a detailed changelog.
- Updated document tags to include "5.3.1", enhancing searchability and relevance.
- Improved document navigation by adding an anchor to the Tyk OAS APIs Compatibility Caveats section.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Addition of 5.3.1 Release Notes and Minor Adjustments</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md
<li>Added tags for version 5.3.1 in the document header.<br> <li> Introduced a new section for the 5.3.1 Release Notes, including <br>release highlights, breaking changes, deprecations, upgrade <br>instructions, downloads, and a detailed changelog.<br> <li> Adjusted the Tyk OAS APIs Compatibility Caveats section to include an <br>anchor for direct linking.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4400/files#diff-14b1be0b2206c9ee21cb1002b55361919932e84558f614c4342f05b8eb5431f8">+98/-2</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi @Eopayemi , Thanks for updating the changelog items to Dashboard PR.
PR reads ok, minor change suggested relating to release summary. When copying over the release summaries from the confluence table does each release summary have every word capitalised?
- Enhanced Dashboard Privacy: Users Search Method Transitioned to POST => Enhanced - Dashboard privacy: Users search method transition to POST

I've had a read through and before release would need:
- peer review
- release date adding ✅ 
- dependencies section added
- download links adding - formatted similar to release notes for 5.3.0 ✅

**Review by dcs3spp — APPROVED** [review]
@andyo-tyk PR LGTM I have added dependencies section with statement that links to dependencies for 5.3.0

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 1]
```diff
@@ -19,6 +19,143 @@ Our minor releases are supported until our next minor comes out.
...
+### Breaking Changes
+**Attention**: Please read this section carefully.
+
+There are no breaking changes in this release, however if moving from an version of Tyk older than 5.3.0 please read the explanation provided with [5.3.0 release]({{< ref "#TykOAS-v5.3.0">}}).
+
+### Deprecations
+There are no deprecations in this release.
+
+### Upgrade Instructions
+If you are using 5.3.0 we advise you to upgrade ASAP and if you are on an older version you should skip 5.3.0 and upgrade directly to this release. Go to the [Upgrading Tyk](#upgrading-tyk) section for detailed upgrade instructions.
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]: If user is skipping 5.3.0, they should take note of the Upgrade Instructions in 5.3.0 too.

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 2]
```diff
@@ -19,6 +19,143 @@ Our minor releases are supported until our next minor comes out.
...
+### Dependencies
+Please refer to [dependencies](#dependencies-5.3.0) provided for release 5.3.0
+
+### Downloads
+- [Docker Image to pull](https://hub.docker.com/r/tykio/tyk-dashboard/tags?page=&page_size=&ordering=&name=v5.3.1)
+- ```bash
+  docker pull tykio/tyk-dashboard:v5.3.1
+  ```
+- Helm charts
+  - [tyk-charts GH Repo](https://github.com/TykTechnologies/tyk-charts/releases)
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]: The latest release notes for Helm Chart is in docs: product-stack/tyk-charts/release-notes/version-1.3

**Review by carolwkc — CHANGES_REQUESTED** [review]

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md (hunk 2 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]: Thanks @caroltyk I have updated. If ok with other comments and changes would you be able to approve?

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]: Thanks @caroltyk I have updated. If ok with other comments and changes would you be able to approve?

**Review by carolwkc — APPROVED** [review]
LGTM

**Pull request merged**